dokka-subprojects/plugin-base/src/test/kotlin/expectActuals/ExpectActualsTest.kt

```diff
@@ -435,4 +439,68 @@ class ExpectActualsTest : BaseAbstractTest() {
             }
         }
     }
+
+    @Test
+    fun `should merge an implicit-expectActual function with a single property with #3685`() = testInline(
+        """
+        /src/common/test.kt
+        expect class Skiko
+
+        /src/jvm/test.kt
+        actual class Skiko actual constructor() {
+             val isShowing = false
+             fun isShowing(): Boolean {
+                return false
+            }
+        }
+        
+        /src/native/test.kt
+        actual class Skiko actual constructor(){
+            fun isShowing(): Boolean {
+                return false
+            }
+        }
+        """.trimMargin(),
+        multiplatformConfiguration
+    ) {
+        pagesGenerationStage = { root ->
+            val cl = root.dfs { it.name == "Skiko" && it is ClasslikePageNode } ?: throw IllegalStateException()
+            assertEquals(2, cl.children.count { it.name == "[jvm]isShowing" })
```

Comment:
Before the fix, there were 2 pages: `[jvm]isShowing` and `isShowing` with the different names, but the same DRI.